docs/index.rst

```diff
@@ -28,8 +37,13 @@ If you're a new programmer, teacher or unsure where to start, begin with the tut
 
 Projects related to MicroPython on the BBC micro:bit include:
 
-* `Mu <https://github.com/ntoll/mu>`_ - a simple code editor for kids, teachers and beginner programmers. Probably the easiest way for people to program MicroPython on the BBC micro:bit.
-* `uFlash <https://uflash.readthedocs.io/en/latest/>`_ - a command line tool for flashing raw Python scripts onto a BBC micro:bit.
+* `micro:bit Python Editor <https://python.microbit.org>`_ - a simple
+   browser-based code editor, designed to help teachers and learners get the
+   most out of text-based programming on the micro:bit.
+* `Mu <https://codewith.mu>`_ - a simple offline code editor for kids, teachers
+   and beginner programmers.
+* `uFlash <https://uflash.readthedocs.io/en/latest/>`_ - a command line tool
+   for flashing raw Python scripts onto a BBC micro:bit.
```

Comment:
Rendering looks off:
![image](https://user-images.githubusercontent.com/29712657/100443419-99045d00-30a1-11eb-800f-7f13740c189c.png)

Reply:
Fixed Should now look like 
![image](https://user-images.githubusercontent.com/31242877/100587296-4963a280-32e8-11eb-89ec-f180fc129cc0.png)